### ISSUE TITLE
"openai migrate" Error: Failed to download Grit CLI

### ISSUE THREAD

**Issue by atthasaeth** [openai/openai-python#1838]
### Confirm this is an issue with the Python library and not an underlying OpenAI API

- [X] This is an issue with the Python library

### Describe the bug

When running the `openai migrate` command, it attempts to download the Grit CLI from `https://github.com/getgrit/gritql/releases/latest/download/marzano-x86_64-unknown-linux-gnu.tar.gz`, but the link appears to be broken or outdated and it response `Error: Failed to download Grit CLI from https://github.com/getgrit/gritql/releases/latest/download/marzano-x86_64-unknown-linux-gnu.tar.gz`

### To Reproduce

1. Install the latest version of openai via pip: pip install openai --upgrade
2. Observe the command attempting to download the file marzano-x86_64-unknown-linux-gnu.tar.gz.

I think that the current Grit releases on GitHub seem to have changed the prefix from marzano- to grit- in the file names, which is likely causing the download to fail.

### OS

Ubuntu 20.04 LTS

### Python version

Python v3.11.4

### Library version

openai 1.53.0

**Comment by mvmendes** [comment]
I could bypass this bug is downloading grit and executing  grit CLI , with "apply openai" parameters:
```
$ openai migrate
Downloading Grit CLI from https://github.com/getgrit/gritql/releases/latest/download/marzano-x86_64-unknown-linux-gnu.tar.gz
Error: Failed to download Grit CLI from https://github.com/getgrit/gritql/releases/latest/download/marzano-x86_64-unknown-linux-gnu.tar.gz

$ curl -fsSL https://docs.grit.io/install | bash
downloading grit 0.1.0-alpha.1730315451 x86_64-unknown-linux-gnu
installing to /home/marcus/.grit/bin
  grit
everything's installed!

To add $HOME/.grit/bin to your PATH, either restart your shell or run:

    source $HOME/.grit/bin/env (sh, bash, zsh)
    source $HOME/.grit/bin/env.fish (fish)

$  source $HOME/.grit/bin/env 

$ grit apply openai
```

**Comment by adenisdeveloper** [comment]
> grit apply openai

Thanks for the tip, it worked! I was having the same problem!

**Comment by albertnieto** [comment]
No news from this issue?

**Comment by tensor-works** [comment]
Same isssue hasn't moved

**Comment by RobertCraigie** [comment]
This will be fixed in [`v1.57.5`](https://github.com/openai/openai-python/pull/1952), thanks to @chiptuned!
